client/src/components/molecules/OngoingAgendaCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Box, Card, Text, Button } from "@/components/atoms";
+import { OngoingAgenda } from "biseo-interface/agenda";
+import {
+  ChoiceComponent,
+  CompletedChoice,
+  NotVotableChoice,
+} from "@/components/molecules";
+import { useAgenda } from "@/services/agenda";
+
+type OngoingAgendaProps = {
```

Comment:
요거도 `interface`로 통일하기
```suggestion
interface OngoingAgendaProps {
```